src/styles/lines/lines.js

```diff
@@ -9,13 +9,17 @@ import {buildPolylines} from '../../builders/polylines';
 import Geo from '../../geo';
 import Utils from '../../utils/utils';
 
+let fs = require('fs');
+let shaderSrc_polygonsVertex = fs.readFileSync(__dirname + '/../polygons/polygons_vertex.glsl', 'utf8');
```

Comment:
Sure, that works. You prefer that to importing the whole `Polygons` symbol and referencing them as:

`vertex_shader_src: Polygons.vertex_shader_src`

Either works for me.